updates/commands.go

```diff
@@ -119,6 +123,111 @@ func createStatusMessage(user users.User, chatId int64, msg tgbotapi.MessageConf
 	return msg
 }
 
+func handleMonthlySummaryCommand(fatBotUpdate FatBotUpdate) {
+	update := fatBotUpdate.Update
+	bot := fatBotUpdate.Bot
+	originalChatId := update.FromChat().ID
+	user, err := users.GetUserFromMessage(update.Message)
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+
+	if user.ID == 0 {
+		return
+	}
+	months := 6 // until we'll let the user enter the number of months
+	start, end := getDuration(months)
+	chatIds, err := user.GetChatIds()
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+	if len(chatIds) > 0 {
+		for _, chatId := range chatIds {
+			createUserMonthlySummaryChart(bot, originalChatId, user, chatId, start, end)
+		}
+	} else {
+		log.Errorf("No chat IDs found for user %s", user.Username)
```

Comment:
Great place to return an error (Golang convention) instead of logging it.
Then log and error centrally (and maybe report to sentry if it makes sense)